charts/tidb-operator/templates/controller-manager-deployment.yaml

```diff
@@ -34,6 +34,8 @@ spec:
         command:
           - /usr/local/bin/tidb-controller-manager
           - -default-storage-class-name={{ .Values.defaultStorageClassName }}
+          - -default-backup-storage-class-name={{ .Values.defaultBackupStorageClassName }}
+          - -tidb-backup-manager-image={{ .Values.tidbBackupManagerImage }}
```

Comment:
What is the process execution model here? How does the controller manager use the backup manager image?

Reply:
This image is mainly used to create backp, restore, and clean jobs to perform the actual backup 、restore and backup data cleanup